client-js/app/elements/labrad-registry.html

```diff
@@ -87,57 +97,53 @@
     }
   </style>
   <template>
-    <div id="reg-menu">
-      <section>
-        <menu-button on-click="newKeyClicked" name="new key" icon-name="create"></menu-button>
-        <menu-button on-click="newFolderClicked" name="new folder" icon-name="folder-open"></menu-button>
-        <menu-button on-click="copyClicked" name="copy" icon-name="content-copy" disabled={{!selected}}></menu-button>
-        <menu-button on-click="renameClicked" name="rename" icon-name="code" disabled={{!selected}}></menu-button>
-        <menu-button on-click="deleteClicked" name="delete" icon-name="delete" disabled={{!selected}}></menu-button>
-      </section>
-    </div>
-    <div id="reg-area">
-      <div id="search-bar">
-        <iron-icon icon="search"></iron-icon>
-        <input type="search" id="search" value="{{filterText::input}}" placeholder="search registry">
+    <div class="z-container">
```

Comment:
is this outer container needed? I don't see any effect if I remove it. I only ask because it would be nice to keep non-semantic markup to a minimum, but not a big deal.